model/Individual/Individual.md

```diff
@@ -0,0 +1,24 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# Individual
+
+## Summary
+
+A profile to be used when sharing individual SPDX `Element`.
+
+## Description
+
+Although most use cases involve sharing `Elements` inside a `Collection`, there are cases where a stand-alone SPDX element may be made available.
+In those cases, the `Element` `creationInfo` property would be required to identify the `Entity` that created the element.
+
+## Metadata
+
+- id: https://rdf.spdx.org/v3/Individual
+- name: Individual
+
+## External properties restrictions
+
+- /Core/creationInfo
+  - type: /Core/Element/CreationInfo
+  - minCount: 1
+  - maxCount: 1
```

Comment:
This is not the right way. Please use (to mark class, property, and only the values changed):

```
- /Core/Element/creationInfo
  - minCount: 1
```

Reply:
So - is the type /Core/CreationInfo?

Reply:
As you don't change the type of the external property, I don't think you have to specify it here (the same goes for maxCount).

Reply:
Right, for external property restrictions, we use _/namespace/class/property_ to point to something and then we list the changed values.

See, for example, how `File` specifies that `name` (a property of `Element`) is mandatory -- this is exactly the same.